src/js/index/todolist.js

```diff
@@ -0,0 +1,121 @@
+const $todoInput = document.querySelector("#new-todo-title");
+const $todoList = document.querySelector("#todo-list");
+const $todoCount = document.querySelector("strong");
+const $filters = document.querySelector(".filters");
+
+$todoInput.addEventListener("keyup", onAddTodoItem);
+$todoList.addEventListener("click", onToggleTodoItem);
+$todoList.addEventListener("dblclick", onEditTodoItem);
+$filters.addEventListener("click", onFilter);
+
+function onAddTodoItem(event) {
+    const todoTitle = event.target.value;
+    const todoList = document.getElementById("todo-list");
+    if (event.key === "Enter" && todoTitle !== "") {
+        todoList.insertAdjacentHTML("beforeend", renderTodoItemTemplate(todoTitle));
+        updateTodoCount();
+        event.target.value = "";
+        changeState();
+    }
```

Comment:
```suggestion
    if (event.key !== "Enter" || todoTitle === "") {
        return;
    }
    
    todoList.insertAdjacentHTML("beforeend", renderTodoItemTemplate(todoTitle));
    updateTodoCount();
    changeState();
    event.target.value = "";
```

early return 하는 것이 가독성이 더 높은것 같아요 :smile: